src/Makefile

```diff
@@ -64,6 +64,9 @@ FLAGS += -DUSE_POLLY
 ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
+ifeq ($(USE_POLLY_ACC),1)
+LLVMLINK += -lPollyPPCG -Wl,-rpath=$(shell $(LLVM_CONFIG_HOST) --libdir),--no-as-needed -lGPURuntime -Wl,--as-needed
```

Comment:
@vtjnash Which file in src/ do you think would be most appropriate to have such an include line ? I was thinking of jitlayers.cpp or codegen.cpp since those files already contain switches related to Polly. But, they didn't seem to set the #include in an appropriate context.